pkgs/applications/networking/sync/unison/default.nix

```diff
@@ -1,12 +1,15 @@
-{stdenv, fetchurl, ocaml, lablgtk, fontschumachermisc, xset, makeWrapper, ncurses
+{stdenv, fetchurl, pkgs, ocaml, lablgtk, fontschumachermisc, xset, makeWrapper, ncurses
 , enableX11 ? true}:
 
+let
+  ocaml = pkgs.ocaml_4_02;
+in
```

Comment:
could you do this in all-packages.nix file? and keep unison expression generic. that way you can easily test with it works with different versions of ocaml